docs/root/configuration/http/http_filters/_include/grpc-json-reverse-transcoder-filter.yaml

```diff
@@ -0,0 +1,59 @@
+admin:
```

Comment:
is admin required for this example?

Reply:
Thanks for pointing out; it's not required, so removed it.